packages/services/cmd/ecs-relay/main.go

```diff
@@ -15,8 +15,9 @@ var (
 	idleTimeoutTime        = flag.Int("idle-timeout-time", 30, "Time in seconds after which a client connection times out. Defaults to 30s")
 	idleDisconnectIterval  = flag.Int("idle-disconnect-interval", 60, "Time in seconds for how oftern to disconnect idle clients. Defaults to 60s")
 	messsageDriftTime      = flag.Int("message-drift-time", 5, "Time in seconds that is acceptable as drift before message is not relayed. Defaults to 5s")
+	minAccountBalance      = flag.Int("min-account-balance", 1000000000000000, "Minimum balance for an account to get its messages relayed. Defaults to 1e15")
```

Comment:
[nit] Also suggest to change to "Defaults to 0.001 ETH" for consistency since in other places like the command line args for faucet (main.go) we use ETH to say what the default is, even though the actual amount is in wei